draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -427,8 +443,8 @@ A zone state reset may be performed by a change of the member node's name (see (
 # Implementation and operational Notes {#implementationnotes}
 
 Although any valid domain name can be used for the catalog name $CATZ, it is
-RECOMMENDED to use either a domain name owned by the catalog producer, or to
-use a name under a suitable Special-Use Domain Name [@!RFC6761].
+RECOMMENDED to use either a domain name controlled by the catalog producer, or
+to use a name under a suitable name such as "invalid." [@!RFC6761].
```

Comment:
I preferred the less specific version (Special-Use)

Reply:
How do you recommend to address the following comment from the AD review:
> Also, what is a "suitable Special-Use Domain Name"? 10.in-addr.arpa.? localhost.? example.com.? Why doesn't this just say that the catalog name MUST be under a domain owned by the catalog producer?

Reply:
Special-Use Domain Names are listed here: https://www.iana.org/assignments/special-use-domain-names/special-use-domain-names.xhtml.
From the names listed in RFC6761, "invalid." is indeed the most suitable. But there may be future Special-Use domain names (".alt" or ".internal") that may be suitable too. If needed we can register something specially for catalog zones, something like "catalog-zones.arpa." maybe.

Reply:
Do we have an answer to Warren's question "Why doesn't this just say that the catalog name MUST be under a domain owned by the catalog producer?"

Reply:
Yes, the catalog zone is not (necessarily) part of the global name space. It is part of the control plane and not the data plane. An unpublished DNS domain (other than to serve the catalog zone) makes therefore sense (at least to me), also to not accidentally leak the zones in the catalog (which may be private).

Reply:
> I preferred the less specific version (Special-Use)

I think "such as" is also not too specific. So I'll go ahead and merge.